bookstore/src/main/webapp/static/css/style.css

```diff
@@ -0,0 +1,281 @@
+.row {
+	padding: 0px;
+	margin: 0px;
+}
+
+.header {
+	background-color: #C0C0C0;
+	height: 150px;
+	border-bottom: 1px solid #e5e5e5;
+}
+
+.header .contact-information {
+	background-color: white;
+	height: 50px;
+	border: 1px solid #dddddd;
+}
+
+.header .contact-information .mail-contact {
+	font-size: 15px;
+	color: #252525;
+	float: right;
+	display: inline-block;
+}
+
+.header .contact-information .phone-contact {
+	float: left;
+	font-size: 14px;
+	color: #252525;
+	display: inline-block;
+}
+
+.header .contact-information .social-network {
+	font-size: 16px;
+	color: #252525;
+	float: right;
+	margin-right: 20px;
+}
+
+.header .contact-information .lang{
+	font-size: 15px;
+	color: #252525;
+	background-color: white;
+	padding-top: 10px;
+}
+
+.header .contact-information .login-top {
+	font-size: 15px;
+	color: #252525;
+	float: right;
+}
+.social-network ul li {
```

Comment:
@danhtrancong 

1. Add a space between line 51, 52 and check for the other lines
2. What is parent class of .social-network